portal-api/src/v1/email/index.test.js

```diff
@@ -0,0 +1,94 @@
+const mockNotifyClientSendEmail = jest.fn();
+
+jest.mock('notifications-node-client', () => ({
+  NotifyClient: jest.fn().mockImplementation(() => ({ sendEmail: mockNotifyClientSendEmail })),
+}));
+
+const sendEmail = require('./index');
+
+describe('sendEmail', () => {
```

Comment:
Thanks for adding these tests @oscar-richardson-softwire, nice improvement to the code :)